src/Dropdown/elements.js

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components';
+
+export const Header = styled.button`
+  display: flex;
+
+  height: 100%;
+  width: 100%;
+
+  padding: 0.8rem 1.2rem;
+
+  cursor: pointer;
+  text-align: left;
+  font-size: ${({ theme: { fonts } }) => fonts.size.medium};
+
+  border: 1px solid ${({ theme: { palette } }) => palette.lightGrey};
+  border-radius: 4px;
+
+  background-color: ${({ theme: { palette } }) => palette.white};
+  box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.08);
+  color: ${({ theme: { palette } }) => palette.darkBlue};
+  &::after {
+    content: '';
+
+    position: absolute;
+    right: 1.4rem;
+    top: 37%;
+
+    width: 0;
+    height: 0;
+
+    border: 0 solid transparent;
+    border-right-width: 3px;
+    border-left-width: 3px;
+    border-bottom: 3px solid ${({ theme: { palette } }) => palette.spaceGrey};
+  }
+  &::before {
+    content: '';
+
+    position: absolute;
+    right: 1.4rem;
+    bottom: 37%;
+
+    width: 0;
+    height: 0;
+
+    border: 0 solid transparent;
+    border-left-width: 3px;
+    border-right-width: 3px;
+    border-top: 3px solid ${({ theme: { palette } }) => palette.spaceGrey};
+  }
+`;
+
+export const Menu = styled.ul`
+  position: absolute;
+  top: calc(100% + 0.4rem);
+  left: 0;
+
+  width: 100%;
+  max-height: 16rem;
+
+  border: 1px solid ${({ theme: { palette } }) => palette.lightGrey};
+  border-radius: 4px;
```

Comment:
Use Theme dimensions radius options instead of `px`

Reply:
this wasn't solved